HelpSource/Classes/EnvironmentRedirect.schelp

```diff
@@ -31,7 +35,13 @@ subsection::redirecting objects
 
 Overriding these methods, one can redirect where objects go when they are assigned to the space. This is done for example in link::Classes/LazyEnvir:: and link::Classes/ProxySpace::.
 
-method::at, put, removeAt, add, linkDoc, unlinkDoc
+method::at, removeAt
```

Comment:
Same as above. Is it the case that you're just listing the methods that are implemented from Environment (above) and that can be overridden (here)? In that case I don't think method:: should be used. It can just be a list. and They should link to the parent implementation's helpfile. 

I also think "linkDoc" and "unlinkDoc" could use some explanation. (I don't know what these do). And all these could be linked to the ProxySpace or LazyEnvir helpfiles.

Reply:
>Same as above.

Answered above.

For other comments:
Ideally, I would like either @telephon or @JordanHendersonMusic to answer.

Reply:
thanks, resolved in https://github.com/supercollider/supercollider/pull/7079/commits/9d6a24c6d8dd7ab7644e0cf9099b3ba93636bce0